certified-connectors/Dynatrace/apiDefinition.swagger.json

```diff
@@ -0,0 +1,982 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Dynatrace",
+    "description": "Dynatrace provides best-in-class observability through an open, AI-powered platform. This connector contains the endpoints for the Dynatrace Problems interface.",
+    "version": "1.0",
+    "contact": {
+      "name": "Connector Support",
```

Comment:
@mandy-swinton please change this one to `Dynatrace Support`, thank you!

Reply:
Updated